packages/devtools/src/schema.ts

```diff
@@ -376,19 +382,18 @@ function defaultFieldSchemaFromFieldDefinition(
       type: 'string',
       title: 'GraphQLDID',
       pattern: "/^did:[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+:[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+$/",
-      maxLength: 80,
+      maxLength: 100,
     }
   }
 
-  // if (fieldTypeIsinstanceOfOrWraps(fieldType, GraphQLStreamReference)) {
-  //   result = {
-  //     ...result,
-  //     type: 'string',
-  //     title: 'CeramicStreamReference',
-  //     maxLength: 80,
-  //     pattern: '<TBD>', //FIXME: define the pattern for StreamReference strings
-  //   }
-  // }
+  if (fieldTypeIsinstanceOfOrWraps(fieldType, CeramicCommitID)) {
+    result = {
+      ...result,
+      type: 'string',
+      title: 'CeramicCommitID',
+      maxLength: 200,
```

Comment:
We will not want to provide the pattern here in the foreseeable future? If we do, maybe let's leave a TODO?

Reply:
I think we might want to eventually, but I'm not sure of the constraints so I think we can leave it empty for now. We'll revisit anyway when we start supporting relations between documents.